### ISSUE TITLE
Beta Banner on current sitesandtrailsbc.ca

### ISSUE THREAD

**Issue by BAAlexK** [bcgov/nr-rec-resources#161]
#### Description:
Add beta banner to direct users to a beta website (aka our MVP) so that we can test and continuously release without disrupting service/access

#### Acceptance Criteria:
- [ ] Add banner to current website
- [ ] Add banner to indicate beta status on new website
- [ ] ...
- [ ] ...

#### Development Checklist:
- [ ] ...
- [ ] ...
- [ ] ...

#### Dependencies
- Blocked by
- Blocking

#### Notes:
- Need to engage with current vendor to be able to do this
- Mobile & desktop designs can be found here: https://www.figma.com/design/TFDtavdpJE2vrB5vEmBB6w/RST---Basic-View?node-id=474-11874&t=ds3xKyv3MwpiFhhc-4
-

**Comment by airliah** [comment]
closing as this is a duplicate of #170 where I've merged info from both tickets together
